diff -Naur a/plugins/build-basic-autotools/plugin.c b/plugins/build-basic-autotools/plugin.c
--- a/plugins/build-basic-autotools/plugin.c	2014-04-26 12:12:08.000000000 -0700
+++ b/plugins/build-basic-autotools/plugin.c	2015-07-30 17:31:51.040327871 -0700
@@ -2142,54 +2142,55 @@
 	gchar *selected;
 	BuildConfiguration *cfg;
 
-	if (phase != ANJUTA_SESSION_PHASE_NORMAL)
-		return;
-
-	configurations = anjuta_session_get_string_list (session, "Build",
-											  "Configuration list");
-
-	build_configuration_list_from_string_list (plugin->configurations, configurations);
-	g_list_foreach (configurations, (GFunc)g_free, NULL);
-	g_list_free (configurations);
+	if (phase == ANJUTA_SESSION_PHASE_NORMAL || phase == ANJUTA_SESSION_PHASE_START )
+	{
 
-	selected = anjuta_session_get_string (session, "Build", "Selected Configuration");
-	build_configuration_list_select (plugin->configurations, selected);
-	g_free (selected);
+		configurations = anjuta_session_get_string_list (session, "Build",
+												  "Configuration list");
 
-	for (cfg = build_configuration_list_get_first (plugin->configurations); cfg != NULL; cfg = build_configuration_next (cfg))
-	{
-		gchar *key;
-		gchar *args;
-		GList *env;
-
-		key = g_strconcat("BuildArgs/", build_configuration_get_name (cfg), NULL);
-		args = anjuta_session_get_string (session, "Build", key);
-		g_free (key);
-		if (args != NULL)
-		{
-			build_configuration_set_args (cfg, args);
-			g_free (args);
-		}
+		build_configuration_list_from_string_list (plugin->configurations, configurations);
+		g_list_foreach (configurations, (GFunc)g_free, NULL);
+		g_list_free (configurations);
+
+		selected = anjuta_session_get_string (session, "Build", "Selected Configuration");
+		build_configuration_list_select (plugin->configurations, selected);
+		g_free (selected);
 
-		key = g_strconcat("BuildEnv/", build_configuration_get_name (cfg), NULL);
-		env = anjuta_session_get_string_list (session, "Build",	key);
-		g_free (key);
-		if (env != NULL)
+		for (cfg = build_configuration_list_get_first (plugin->configurations); cfg != NULL; cfg = build_configuration_next (cfg))
 		{
-			GList *item;
+			gchar *key;
+			gchar *args;
+			GList *env;
+
+			key = g_strconcat("BuildArgs/", build_configuration_get_name (cfg), NULL);
+			args = anjuta_session_get_string (session, "Build", key);
+			g_free (key);
+			if (args != NULL)
+			{
+				build_configuration_set_args (cfg, args);
+				g_free (args);
+			}
 
-			/* New variables are added at the beginning of the list */
-			for (item = env; item != NULL; item = g_list_next (item))
+			key = g_strconcat("BuildEnv/", build_configuration_get_name (cfg), NULL);
+			env = anjuta_session_get_string_list (session, "Build",	key);
+			g_free (key);
+			if (env != NULL)
 			{
-				build_configuration_set_variable (cfg, (gchar *)item->data);
-				g_free (item->data);
+				GList *item;
+
+				/* New variables are added at the beginning of the list */
+				for (item = env; item != NULL; item = g_list_next (item))
+				{
+					build_configuration_set_variable (cfg, (gchar *)item->data);
+					g_free (item->data);
+				}
+				g_list_free (env);
 			}
-			g_list_free (env);
+
 		}
 
+		build_project_configured (G_OBJECT (plugin), NULL, NULL, NULL);
 	}
-
-	build_project_configured (G_OBJECT (plugin), NULL, NULL, NULL);
 }
 
 static void
