diff -aur firefox-91.3.0_orig/python/mozbuild/mozbuild/frontend/gyp_reader.py firefox-91.3.0/python/mozbuild/mozbuild/frontend/gyp_reader.py
--- firefox-91.3.0_orig/python/mozbuild/mozbuild/frontend/gyp_reader.py	2021-08-07 07:39:32.616362144 +0000
+++ firefox-91.3.0/python/mozbuild/mozbuild/frontend/gyp_reader.py	2021-08-07 07:39:03.680361110 +0000
@@ -330,6 +330,8 @@
                         resolved
                     ):
                         continue
+                    if include == "%":
+                        continue
                 context["LOCAL_INCLUDES"] += [include]
 
             context["ASFLAGS"] = target_conf.get("asflags_mozilla", [])
